php/libraries/Visit.class.inc

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Describe a LORIS visit
+ *
+ * PHP Version 7
+ *
+ * @category Model
+ * @package  Visits
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS;
+
+/**
+ * Representation of a Visit in LORIS
+ *
+ * @category Model
+ * @package  Visits
+ * @author   Xavier Lecours Boucher <xavier.lecoursboucher@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Visit
+{
+    /**
+     * The visit ID
+     *
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * The visit name (formely known as visit_label)
+     *
+     * @var string
+     */
+    protected $visit;
```

Comment:
maybe call the variable `$name` ?? 

the `$visit` variable in the Visit class is not useful